yarn.lock

```diff
@@ -2182,6 +2182,31 @@ babel-code-frame@6.26.0:
     esutils "^2.0.2"
     js-tokens "^3.0.2"
 
+babel-core@^6.0.0, babel-core@^6.26.0:
```

Comment:
This lock is pulling in Jest 23 and Babel 6. Is that correct?

Reply:
"correct" is relative 😛: https://github.com/facebook/jest/pull/7707#discussion_r251068967

Reply:
That's unfortunate!